docs/install-manual.md

```diff
@@ -27,7 +27,7 @@ You can run Docker Swarm on Linux 64-bit architectures. You can also install and
 
 Take a moment and identify the systems on your network that you intend to use. Ensure each node meets the requirements listed above.
 
-
+> **Warning**: Before going any further with the setup process, make sure that Docker is not pre-installed on your VM image and/or that the machine is not created using VM cloning. Docker generates a unique ID for the Engine that is located in the `/etc/docker/key.json` file. If a VM is the result of cloning with a docker daemon pre-installed, Swarm will not be able to recognize remote docker Engines as different. This is because the same unique ID is duplicated on those Engines from the VM cloning. As a result, you will see a single docker Engine registered on Swarm. As a workaround you can stop the daemon on those machines, delete the `/etc/docker/key.json` and finally restart the Engines. This will regenerate a new ID for each of those docker daemons.
```

Comment:
@moxiegirl Yeah I admit I went a little too overboard with that warning :sweat_smile: 

Will try to re-structure and make it more clear and concise.